src/game/components/GameToyClient.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import { GamePhase, useGameValue } from '../GameProvider';
+import { useAutoRef, useToyClientValue } from '../../utils';
+
+export const GameVibrator = () => {
```

Comment:
You have renamed the file to "GameToyClient", but the component is still called "GameVibrator".
We should rename this appropriately too, then.

Reply:
Fixed in [5f193d0](https://github.com/clynamic/joi.how/pull/18/commits/5f193d0e0576e703b16dc4d357f672ad7c6f6a33)